server/server.js

```diff
@@ -88,6 +88,7 @@ function makeServer() {
     'layout',
     'page',
     'blog',
+    'timeline',
```

Comment:
It *should* get this from the `templates` you pass to `husky.registerPage` but I seem to have passed them here as well, did it not work if you didn't include it here?

Reply:
removed them and seems to still work.